version.props

```diff
@@ -3,8 +3,9 @@
     <AspNetCoreMajorVersion>3</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleaseLabel>preview</PreReleaseLabel>
-    <PreReleaseBrandingLabel>Preview</PreReleaseBrandingLabel>
+    <PreReleasePreviewNumber>3</PreReleasePreviewNumber>
+    <PreReleaseLabel>preview$(PreReleasePreviewNumber)</PreReleaseLabel>
+    <PreReleaseBrandingLabel>Preview $(PreReleasePreviewNumber)</PreReleaseBrandingLabel>
```

Comment:
Curious: Why use a variable here and not in the other repos? Is this because we have both `$(PreReleaseLabel)` and `$(PreReleaseBrandingLabel)` here?

Reply:
Yes. One fewer thing that can get out of sync.

Reply:
PreReleaseBrandingLabel is for the "human friendly" version strings in installers, like "ASP.NET Core 3.0.0 Preview 3"